[WIP] Add donation branch selector

TomSesselmann opened the pull request:
<img width="975" alt="screen shot 2017-09-11 at 10 41 31 pm" src="https://user-images.githubusercontent.com/23013276/30274871-99003de8-9742-11e7-95e7-39f83a1a6f21.png">

Still needs some styling and to actually connect to the Paypal button.

TomSesselmann committed:
Add jurisdictions toName function

TomSesselmann committed:
attempt to add selector

TomSesselmann committed:
Create list of states because you can’t iterate over Union Types

TomSesselmann committed:
Fix type mismatch

ghost commented:
Your pull request doesn't follow our guidelines. Please fix the following:

* Pull request title must be in imperative, present tense \(e\.g\. "add", "fix", "change"\) [(?)](https://gitmagic.io/rules/#/pull-request/subject-must-be-in-tense)

[Click here](https://app.gitmagic.io/voteflux/vote-flux-v2/53/72c10fe5-5217-412a-b376-32a0e80ee5ee/b25a1240-96e3-4d74-bdba-8d8cc5a97300) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

TomSesselmann commented:
I do admit, the compile errors are nice 😀
Still working my way around the syntax though

ghost commented:
Thank you, the title and description now looks good! :bowtie:

XertroV committed:
Make dropdown update model

- Also fixed some minor css issues
- Refactor Flux.Jurisdictions to be a bit nicer, but less type safe :(
